### PR TITLE
Add new recipe: deblur

### PR DESCRIPTION
**Pull request by thermokarst**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

---

This recipe is a psuedo-port from the [biocore/conda-recipes](https://github.com/biocore/conda-recipes/tree/master/recipes/deblur) repo (minor modifications made to match linting and other bioconda requirements).

Thanks!

### CODE REVIEW

File: recipes/deblur/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name: deblur
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/deblur/archive/{{ version }}.tar.gz
+  sha256: bb43fefbf9f01db53a1f3bdff755c006a22717f976c8fa3fa242f74cb85380f4
+
+build:
+  number: 0
+  skip: True # [py27,py34]
```
Comment (druvus) [inline recipes/deblur/meta.yaml]: Bioconda is not building for py34 so it could be removed.

File: recipes/deblur/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - setuptools
+    - scikit-bio >=0.5.0
+    - vsearch ==2.0.3
+    - mafft ==7.310
+    - sortmerna ==2.0
+    - biom-format
+    - h5py
+    - click
+  run:
+    - python 3.5*
```
Comment (druvus) [inline recipes/deblur/meta.yaml]: Please remove the python pinning here and only use the skipping in build section.

File: recipes/deblur/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - biom-format
+    - setuptools
```
Comment (druvus) [inline recipes/deblur/meta.yaml]: Is setuptools needed as a run dependency?

Comment (wasade) [reply recipes/deblur/meta.yaml]: Most likely not

Comment (thermokarst) [reply recipes/deblur/meta.yaml]: Great, thanks! I will remove it.

**Comment by thermokarst** [comment]
Looking into CI failures right now - these aren't showing up when I run the circleci client locally.

**Comment by daler** [comment]
@thermokarst you can now try the new `bootstrap.py` method for local testing, should pick up these issues by running the mulled-build test.

Using the latest from master branch:

```bash
./bootstrap.py /tmp/miniconda
source ~/.config/bioconda/activate
bioconda-utils build recipes config.yml --docker --mulled-test --git-range master
```

**Comment by thermokarst** [comment]
Thanks for the review @druvus! I made the changes you requested, and I also enabled the `extended-base` container, since some of the indirect dependencies implicated in this package [use `libGL`](https://bioconda.github.io/troubleshooting.html#using-the-extended-image). Please let me know if there is anything else I can do. Thanks so much!

**Comment by thermokarst** [comment]
Thanks for adding those additional recipe attributes, @druvus. Apologies if I missed that somewhere in the recipe checklist.

**Comment by druvus** [comment]
thanks @thermokarst

**Pull request merged**